frontend/src/app/globals.css

```diff
@@ -16,6 +15,9 @@
   background-color: var(--background);  
 }
 
+  background-color: var(--background);  
```

Comment:
I think this is extra, and causing the build to break due to the additional }.

Reply:
I think this was part of a weird conflict I solved that kept on coming up even after I made the fix. It's now gone.